etna/datasets/tsdataset.py

```diff
@@ -900,21 +900,20 @@ def update_columns_from_pandas(self, df: pd.DataFrame, regressors: Optional[List
         if regressors is not None:
             self._regressors = list(set(self._regressors) | set(regressors))
 
-    def remove_columns(self, columns: List[str]):
+    def remove_features(self, features: List[str]):
```

Comment:
Why have we changed it?

Reply:
We decided to rename the parameter "columns" -> "features" in to_pandas/to_flatten because this parameter is about the features, not the technical columns like "segment" and "timestamp". Here I decided to do the same to make them consistent